.mailmap

```diff
@@ -42,10 +115,19 @@ Marco Hugentobler <marco.hugentobler@sourcepole.ch> <marco@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <marco.hugentobler@sourcepole.ch>
 Marco Hugentobler <marco.hugentobler@sourcepole.ch> mhugent <mhugent@c8812cc2-4d05-0410-92ff-de0c093fc19c>
 
+Mario Baranzini <mario@opengis.ch> <marioba@users.noreply.github.com>
```

Comment:
```suggestion
Mario Baranzini <mario.baranzini@gmail.com>
```

Reply:
The two e-mail accounts included in the mapping are found in the commit history. That's why they should appear in the file.

They are now mapped to `Mario Baranzini <mario.baranzini@gmail.com>` in [679036d](https://github.com/gacarrillor/QGIS/commit/679036d7b27a00626c0acb8870674a1feb5bcda7)